swagger.yaml

```diff
@@ -69,17 +68,203 @@ paths:
                 type: array
                 items:
                   $ref: "#/components/schemas/Tasks"
-  
-   
+
+  "/salesLeads":
+    get:
+      operationId: listSalesLeads
+      summary: Lists all Leads from Pipedrive
+      description: |
+        Lists Leads from Pipedrive
+      tags:
+        - Leads
+      responses:
+        "200":
+          description: Lists Leads from Pipedrive
+          content:
+            application/json;charset=utf-8:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Lead"
+
+  "/salesDeals/{status}":
+    get:
+      operationId: listSalesDeals
+      summary: Lists all Deals from Pipedrive
+      description: |
+        Lists open Deals from Pipedrive
+      tags:
+        - Deals
+      parameters:
+        - name: status
+          in: path
+          required: true
+          schema:
+            type: string  
+          description: Pipedrive Deal status    
+      responses:
+        "200":
+          description: Lists open Deals from Pipedrive
```

Comment:
```suggestion
          description: Lists Deals from Pipedrive
```

Reply:
OR should this list open deals, or does it depend on the type of status passed in for the request. You could set a default status if no status is passed, or have no options to pass any status and have open as the default. Whatever you think is best but I think this needs to be clarified.